variables.tf

```diff
@@ -879,17 +868,9 @@ variable "hms_housekeeper_additional_environment_variables" {
s_threads_waiting_count_value", "java_lang_memory_heapmemoryusage_used", "metrics_memory_heap_init_value", "metrics_api_get_partition_by_name_count", "metrics_api_get_partitions_by_names_count", "metrics_api_get_partition_names_count", "metrics_api_get_partitions_by_expr_count", "metrics_api_get_partitions_count", "metrics_api_get_partition_count", "metrics_api_get_partitions_by_filter_count",  "metrics_api_get_partitions_by_filter_50thpercentile", "metrics_api_get_partitions_by_filter_95thpercentile", "metrics_api_get_partitions_by_filter_999thpercentile", "metrics_api_add_partitions_count", "metrics_api_add_partitions_req_count", "metrics_api_drop_partition_by_name_count", "metrics_api_add_partition_count", "metrics_api_alter_partitions_count", "metrics_api_create_table_count", "metrics_api_alter_table_with_cascade_count", "metrics_api_get_table_meta_count", "metrics_api_get_table_metas_count", "metrics_api_get_table_count", "metrics_api_alter_table_count", "metrics_api_get_tables_count", "metrics_api_get_all_tables_count", "metrics_api_drop_table_count", "metrics_api_get_multi_table_count", "metrics_api_get_database_count", "metrics_api_get_all_databases_count", "metrics_api_get_databases_count", "metrics_api_create_function_count", "metrics_api_getmetaconf_count", "metrics_api_alter_table_with_environment_context_count", "metrics_api_delete_column_statistics_by_table_count", "metrics_api_get_functions_count", "metrics_api_get_function_count", "metrics_api_shutdown_count", "metrics_api_flushcache_count", "metrics_api_get_indexes_count", "metrics_api_get_config_value_count", "metrics_api_set_ugi_count", "metrics_api_get_all_functions_count", "metrics_api_get_table_req_50thpercentile", "metrics_api_get_table_req_95thpercentile", "metrics_api_get_table_req_999thpercentile", "metrics_api_get_table_req_count", "metrics_api_get_table_req_max", "metrics_api_get_databases_count", "metrics_api_get_databases_50thpercentile", "metrics_api_get_databases_95thpercentile", "metrics_api_get_databases_999thpercentile", "metrics_api_get_databases_max", "metrics_api_get_partitions_50thpercentile", "metrics_api_get_partitions_95thpercentile", "metrics_api_get_partitions_999thpercentile", "metrics_api_get_partitions_count", "metrics_api_get_partitions_max", "metrics_api_get_partitions_50thpercentile", "metrics_api_get_partitions_95thpercentile", "metrics_api_get_partitions_999thpercentile", "metrics_api_get_database_50thpercentile", "metrics_api_get_database_95thpercentile", "metrics_api_get_database_999thpercentile", "metrics_kafka_listener_failures_count", "metrics_kafka_listener_successes_count", "metrics_api_get_table_objects_by_name_req_max", "metrics_open_connections_count", "java_lang_memory_heapmemoryusage_max", "metrics_memory_non_heap_used_value", "metrics_memory_non_heap_max_value", "java_lang_garbagecollector_lastgcinfo_duration", "metrics_jvm_pause_warn_threshold_count", "metrics_jvm_pause_extrasleeptime_count", "metrics_threads_deadlock_count_value"]
```

Comment:
It would be better to keep the existing formatting of one metric per line to improve readability.

Reply:
Done!

Reply:
Thanks!